huaweicloud/resource_huaweicloud_cce_cluster_v3.go

```diff
@@ -323,7 +327,12 @@ func resourceCCEClusterV3Read(d *schema.ResourceData, meta interface{}) error {
 	d.Set("security_group_id", n.Spec.HostNetwork.SecurityGroup)
 	d.Set("region", GetRegion(d, config))
 
-	cert, err := clusters.GetCert(cceClient, d.Id()).Extract()
+	r := clusters.GetCert(cceClient, d.Id())
+	kubeConfigRaw, _ := jsonMarshal(r.Body)
```

Comment:
the error result should be judged here if the r.Body is abnormal.

Reply:
done